tike/coverage.py

```diff
@@ -108,9 +110,11 @@ def _coverage_interface(object_grid, object_corner, object_size,
             probe_grid, probe_size, theta, v, h)
 
 
-def coverage(object_grid, object_corner, object_size,
-             probe_grid, probe_size, theta, v, h,
-             dwell=None, **kwargs):
+def coverage(
+    object_grid, object_corner, object_size,
```

Comment:
More indentation needed to distinguish it from function level.

Reply:
No. Read the message from 5354146b4ac25c8e9fb68aa3be65fb8f2c764993.

Reply:
See: https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
PEP8 allows this:
```python
result = some_function_that_takes_arguments(
    'a', 'b', 'c',
    'd', 'e', 'f',
)
```
which I think is what I did. The reason why I changed the indentation is because of the folding difference between A and B.

```python
# Unfolded A: What we had before.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    """Comment."""
    print(var_one)

# Folded A:
def long_function_name( ...

# Unfolded B: My update.
def long_function_name(
    var_one, var_two, var_three,
    var_four
):
   """Comment.""""
    print(var_one)

# Folded B
def long_function_name(
    var_one, var_two, var_three,
    var_four
): ...
```

I don't think another indentation level is necessary because of the break for the comment and the function definition ends with `):` which is one level above the function level. If you're still not convinced, I could also be happy with option C:
```python
# Unfolded C:
def long_function_name(
        var_one, var_two, var_three,
        var_four
):
   """Comment.""""
    print(var_one)

# Folded C
def long_function_name(
        var_one, var_two, var_three,
        var_four
): ...
```

Reply:
C please.